src/inc/til/rect.h

```diff
@@ -0,0 +1,851 @@
+// Copyright (c) Microsoft Corporation.

```

Comment:
(note for reviewers) Use a diff tool (I used text-compare.com) to compare rect.h to rectangle.h and make your life easier haha.

Reply:
It's probably better in general to check these PRs out locally and compare them VS Code for instance.
BTW VS Code has this which is nice:
![image](https://user-images.githubusercontent.com/2256941/145863775-3ff5a76b-1a0b-4512-9b33-d16f76636c4e.png)